Lua.md

```diff
@@ -2,7 +2,6 @@
 
 ## Contents
 1. [register](#register)
-2. [trview](#trview)
 3. [camera](#camera)
```

Comment:
```suggestion
2. [camera](#camera)
```

Reply:
I can just delete this whole file actually